tests/x11regressions/libreoffice/libreoffice_double_click_file.pm

```diff
@@ -39,6 +39,7 @@ sub run() {
         if ($tag ne 'xlsx') {
             hold_key "alt";
             send_key_until_needlematch("libreoffice-nautilus-window", "tab");
+            wait_still_screen;
```

Comment:
LGTM.
And the default stilltime of wait_still_screen is 7s which is a little bit longer?
shorter time may work as well: wait_still_screen 3;